notebooks/text_models/solutions/keras_for_text_classification.ipynb

```diff
@@ -176,7 +176,7 @@
    "cell_type": "markdown",
```

Comment:
<p><code>Keras Tokenizer class</code> is not used anymore. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Updated the text to remove keras tokenizer class and also rephrased a few sentences. Thanks for catching this! </p>